reference/compiling_for_android.rst

```diff
@@ -215,14 +198,14 @@ failure should be presented there.
 
 Seek assistance if you can't figure it out.
 
-Application exits immediately
+2.) Application exits immediately
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If the application runs but exits immediately, there might be one of the
 following reasons:
 
--  libgodot_android.so is not in ``libs/armeabi``
+-  libgodot_android.so is not in ``lib/armeabi-v7a``
 -  Device does not support armv7 (try compiling yourself for armv6)
--  Device is Intel, and apk is compiled for ARM.
+-  Device is ARM, and apk is compiled for intel.
```

Comment:
Sorry, late to reply (been having a lunch time)

It's not about which one dominate the market. Intel devices (which of course powered with x86 cpu) CAN run apk which compiled for ARM devices. On opposite side, ARM devices CAN'T run apk which compiled for Intel devices.

So, Device is Intel, and apk is compiled for ARM == Not a trouble, as Intel devices could be able to run Android application which build for ARM arch.

:-)